lib/fprotect/fprotect_rram.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <errno.h>
+#include <soc.h>
+#include <nrf_erratas.h>
+#include <hal/nrf_rramc.h>
+
+#define kB_Pos (10)
+#define REGION_SIZE ((RRAMC_REGION_CONFIG_SIZE_Msk >> RRAMC_REGION_CONFIG_SIZE_Pos) << kB_Pos)
```

Comment:
It's not obvious to this code reader that RRAMC REGION SIZE unit is a KB.

```suggestion
#define RRAMC_REGION_CONFIG_SIZE_UNIT 1024 /* bytes */
#define REGION_SIZE ((RRAMC_REGION_CONFIG_SIZE_Msk >> RRAMC_REGION_CONFIG_SIZE_Pos) * RRAMC_REGION_CONFIG_SIZE_UNIT)
```